### ISSUE TITLE
Mac instance has dependency on Homebrew

### ISSUE THREAD

**Issue by bmcminn** [huytd/dev.env#1]
Your documentation fails to mention the default `apps/` shell files utilize `brew install <APPNAME>`, which not everyone has installed. Granted most devs have probably installed it at one point or another, but this creates issues for other platforms and discounts your notion of "zero config" environments.

For Debian Linux users, using `apt-get` is standard fare, but Windows users who don't have something like [Chocalatey](https://github.com/chocolatey/choco) installed are out of luck with this utility.

Curious if you've considered writing a standalone install script that downloads and installs the binary into a standardized directory?

**Comment by huytd** [comment]
yeah you're right. actually I put the code to automatically install `brew` in `install.sh`

I will update the code to use `apt-get` or `yum` if the current OS is Linux. I will hold back the Windows support at this moment. 

At the beginning I was created the standalone script for `dev`, but later, I decided to split it to smaller scripts (in `apps` and `env` folders) for easier integration more languages/apps

**Comment by bmcminn** [comment]
:+1: Dig it.

I use a Macbook at work, but have a Windows machine at home, so I may tinker around with testing/building some Win utilities for this. Seems like a good project :)

**Comment by lackerman** [comment]
@bmcminn You could use [chocolatey](https://chocolatey.org/) for Windows.

**Comment by bmcminn** [comment]
@lackerman I think that's the plan for this, though the `install.sh` or `install.cmd` file should be able to determine the OS environment and prompt the user to install a package manage like Homebrew or Chocolatey provided an instance wasn't already discovered.

**Comment by lackerman** [comment]
@bmcminn Fair enough. It makes sense.
